onboarding-manager/pkg/cloudinit/infra.cfg

```diff
@@ -272,4 +322,22 @@ datasource:
                 systemctl stop sshd
                 systemctl disable sshd
                 {{- end }}
+                sudo tee /etc/logrotate.d/ufw << 'END'
+                /var/log/ufw.log
+                {
+                  rotate 4
+                  weekly
+                  missingok
+                  notifempty
+                  size 1G
+                  compress
+                  delaycompress
+                  sharedscripts
+                  postrotate
+                  [ -x /usr/lib/rsyslog/rsyslog-rotate ] && /usr/lib/rsyslog/rsyslog-rotate || true
+                  endscript
+                }
+                END
+                sudo systemctl restart ufw.service
+                sudo ufw logging low
```

Comment:
This two lines must only be executed if ufw is used (for Ubuntu only). See line 248 and onwards.